dom/src/test/java/org/apache/james/mime4j/field/LenientContentDispositionFieldTest.java

```diff
@@ -120,35 +113,25 @@ public void testGetFilename() throws Exception {
 
     @Test
     public void testGetFilenameEncoded() throws Exception {
-        byte[] data = ("Content-Disposition: attachment;\n" +
-            " FileName=\"=?WINDOWS-1251?Q?3244659=5F=C0=EA=F2_=E7=E0_=C8=FE=EB=FC_?=\n" +
-            " =?WINDOWS-1251?Q?2020.pdf?=\"")
-            .getBytes(StandardCharsets.UTF_8);
-
-        ContentDispositionField f = parse(data);
+        ContentDispositionField f = parse("Content-Disposition: attachment;\n" +
+        " FileName=\"=?WINDOWS-1251?Q?3244659=5F=C0=EA=F2_=E7=E0_=C8=FE=EB=FC_?=\n" +
+        " =?WINDOWS-1251?Q?2020.pdf?=\"");
```

Comment:
Hard for me to tell but it looks like only indent change.

THis makes reviewing the diff harder...

Reply:
I try to fix the ident.
What's really the difference here is that I now use the old/existing parse() Method that takes a String instead of the one introduced for those tests that takes a byte array. (see removed parse Method in this test class)